lib/api/core/hotelClerk.js

```diff
@@ -287,25 +291,26 @@ class HotelClerk {
    * @return {Object}
    */
   removeRooms (request) {
-    if (!this.kuzzle.realtime.exists(request.input.resource.index, request.input.resource.collection)) {
-      errorsManager.throw('internal', 'hotelclerk', 'subscription_not_found', request.input.resource.index, request.input.resource.collection);
+    const { index, collection } = request.input.resource;
+
+    if (!this.kuzzle.realtime.exists(index, collection)) {
```

Comment:
Removing a non-existing room should be a no-op operation instead of throwing an error.